assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -129,6 +129,9 @@ spec:
 {{- if .KPInfo }}
         - "--encryption-provider-config=/etc/kubernetes/kms-config/config.yaml"
 {{- end }}
+{{- if .KonnectivityEnabled }}
+        - "--egress-selector-config-file=/etc/kubernetes/config/egress-config.yaml"
```

Comment:
Where is the setup for this config?  I think this needs to be in this PR if referenced.

Reply:
It is part of the kube-apiserver configmap. Now I've added to the toolkit too. But please note, in case of ROKS, we apply this configmap from our sources. Thank you!